CHANGELOG.rst

```diff
@@ -2,6 +2,14 @@
 Changelog
 =========
 
+Version 15.6
+============
+
+* Added requirements lock file that is automatically generated with tox and pre-commit.
```

Comment:
requirements.txt seems not used anywhere?

Reply:
Or how is the vulnerability scanner configured to use the lockfile? Either way if it's only used by CI and not even included in the published python package I don't think a changelog is needed

Reply:
I asked for one to track these changes in case some external dev runs into them. As we often have externals making PRs on this repo.

Reply:
The requirements.txt is used by the Aikido app integration and hence will not be visible here. And, the changelog is added as @Aerylia asked for it.

Reply:
is requirements.txt part of the published python package?

Reply:
Well perhaps it doesn't matter but I'd consider leaving it out to avoid confusion

Reply:
Ok, it can be removed.

Reply:
Reverted. We can squash the commits while merging to make it clean.

Reply:
I don't see any benefit for having this lockfile if the python package and CI are not using it. I think we should either make the package depend on the lockfile, although for client libraries this wouldn't make much sense and would actually be a big nuisance for runtimes depending on this library. The other option is to simply ignore this aikido complaint which I think is correct for client library repos.